fmriprep/workflows/epi.py

```diff
@@ -233,7 +238,12 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                 ('outputnode.itk_epi_to_t1', 'inputnode.itk_epi_to_t1')]),
             (epi_mni_trans_wf, outputnode, [('outputnode.epi_mni', 'epi_mni'),
                                             ('outputnode.epi_mask_mni', 'epi_mask_mni')]),
-        ])
+                        ])
+        if use_aroma:
+            workflow.connect([
+                (epi_mni_trans_wf, discover_wf, [('outputnode.epi_mask_mni', 'inputnode.epi_mask_mni'),
+                                                 ('outputnode.epi_mni', 'inputnode.epi_mni')])
+                            ])
```

Comment:
Similarly, this doesn't need its own `if` block. Unused connections are safe to make.